packages/vapor/resources/icons/svg/variant.svg

```diff
@@ -0,0 +1,6 @@
+<svg width="56px" height="56px" viewBox="0 0 56 56" version="1.1" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Again, the svg could be cleaned up. Something like the following doesn't change the look of the icon and removed the fixed `width` and `height` so that we can later set its size using css classes.

```html
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 56 56">
  <circle cx="27" cy="27" r="26.0357143" fill="none" stroke="#94A9BF" transform="translate(1 1)"/>
  <path fill="#94A9BF" d="M36.0339074 17l4.3780953 3.1623025v3.7301516c0 .6403458-.6046526 1.1802156-1.4223386 1.3365689.0285494.1228486.0432272.2505284.0432272.3814682v11.6083459c0 .982049-.825628 1.7811629-1.8411137 1.7811629H18.8082224c-1.0154857 0-1.8411137-.7991139-1.8411137-1.7811629V25.6104912c0-.1309398.0146778-.2586196.0425265-.3815808-.8169853-.1562407-1.4216379-.6961105-1.4216379-1.3364563v-3.7301516L19.9660926 17h16.0678148zm1.4084481 8.2737776h-18.884711c-.1291376.0737039-.2114244.2014049-.2114244.3367136v11.6083459c0 .2101573.1983246.4020515.4620023.4020515h18.3835552c.2636777 0 .4620023-.1918942.4620023-.4020515V25.6104912c0-.1353087-.0822868-.2630097-.2114244-.3367136zm-6.5433254 1.3800023c1.0628812 0 1.9278598.7422377 1.9278598 1.6541062v.8291218c0 .9118684-.8649786 1.6541062-1.9278598 1.6541062h-5.7980602c-1.0628812 0-1.9278598-.7422378-1.9278598-1.6541062v-.8291218c0-.9118685.8649786-1.6541062 1.9278598-1.6541062h5.7980602zm-.1435641 1.3791114l-5.5127699.0013763-.0009189 1.3777351 5.5164456-.0013764-.0027568-1.377735zm8.2774253-6.8955571H16.9671087v2.6221791c.1159496.0790561.3086126.1360438.5313822.1360438l.8174172-.0011828c.1567507-.0420965.3218578-.0645998.4923143-.0645998h18.3835552c.1704565 0 .3355636.0225033.4923143.0645998l.8178945.0011828c.2225141 0 .4150291-.0569663.5309049-.1360062v-2.6222167zm-3.4449646-2.7582228H20.4120733L18.5029973 19.758h18.993l-1.9080706-1.3788886z"/>
</svg>
```

Reply:
We could go even further and remove the `fill` and `stroke` attributes on the `circle` and `path` tags. That way we could see the icon to whatever color we see fit from [our color palette](https://vapor.cloud.coveo.com/#/components/Color)

Reply:
I made the changes in regards to `width`, `height` and `version`.  For the `fill` and `strokes`, I tried removing them separately and each had an impact on the image.
![image](https://user-images.githubusercontent.com/61028694/78922209-2be57e00-7a64-11ea-94ce-e942a8439586.png)
![image](https://user-images.githubusercontent.com/61028694/78922236-33a52280-7a64-11ea-940a-b464583f8cea.png)
![image](https://user-images.githubusercontent.com/61028694/78922253-3a339a00-7a64-11ea-9445-b7550ec977bd.png)